src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/integration/IntegrationTestBase.kt

```diff
@@ -115,3 +149,44 @@ abstract class IntegrationTestBase {
       .expectBody(Long::class.java)
       .returnResult().responseBody!!
 }
+
+@TestConfiguration
+class TestEmailConfiguration {
+  @Bean
+  fun emailService() =
+    EmailService { email ->
+      when (email) {
```

Comment:
Could this be another file on its in the test/config package, rather than in the integration test base class? just a thought.